proposals/lip-0048.md

```diff
@@ -25,15 +25,34 @@ This LIP defines the fee system in a modular way, as currently [used in the Lisk
 
 ## Rationale
 
+The fee structure presented here serves to allow defining sufficient cost to manage state growth and prevent transaction pool spamming. In principle, any transaction resulting in a state change should incur a fee. This reasoning is implemented in the `payFee` method, which can be invoked by other modules.
+
```

Comment:
I would suggest the following structure:

Subsection: Transaction fees. Include the "fee token ID" and "Minimum Fee Per Transaction" sub-sub sections.

Subsetion: CCM fees.

Then the discussion on fee pool applies to both trs and ccm fees. 

Also, we could then expalin a bit more in detail the `payFee` functionality, perhaps with one example of a transaction when it is called (e.g., validator registration) and one example of a ccm processing when it is called (e.g., need to initialize a user account).

Reply:
Noted.